### DISCUSSION TITLE
perf_graph_reporter tests fail

### DISCUSSION THREAD

**Discussion by hzclarksm** [idaholab/moose#24049] [Q&A Getting Started]
perf_graph_reporter tests fail because python2 is used instead of python3.  A simple fix is

diff --git a/test/tests/reporters/perf_graph_reporter/tests b/test/tests/reporters/perf_graph_reporter/tests
index 8dd72564b0..f28107debd 100644
--- a/test/tests/reporters/perf_graph_reporter/tests
+++ b/test/tests/reporters/perf_graph_reporter/tests
@@ -12,7 +12,7 @@
     []
     [verify]
       type = RunCommand
-      command = 'python verify_perf_graph_reporter.py'
+      command = 'python3 verify_perf_graph_reporter.py'
       prereq = test/run
       detail = 'and shall provide a tool to post process said information'
     []
@@ -39,7 +39,7 @@
     []
     [verify]
       type = RunCommand
-      command = 'python verify_perf_graph_reporter.py recover'
+      command = 'python3 verify_perf_graph_reporter.py recover'
       prereq = recover/run
       detail = 'and shall provide a tool to post process said information'
     []

**Comment by milljm** [comment]
Will address in https://github.com/idaholab/moose/issues/24061
